CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
 
 namespace CompanyEmployees.Core.Domain.Entities
```

Comment:
Consider making the `Salary` property nullable (`decimal?`) if it's optional or not always set. This aligns with the nullable reference types enabled in the project and improves clarity.

```suggestion
public decimal? Salary { get; set; }
```